src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> The items are on the very same level as the headings. You do not have a nesting.

My point was, that the items need at least a separate DIV around them, which is usually adressed for the animation.

> Where does the requirement that only one group is opened at a time suddenly come from? 

From ILIAS. This is an existing behaviour for accordions.

> Where does the requirement to have multiple nesting of groups suddenly come from?

From the existing nesting of things like panels, lists and now groups.

> Just like with the heading, you could have a level parameter on the group, pass it from the list, pass it down to other groups, etc.

Of course I could. But it is not a property of the group, but of the heading. Means:  A heading component would need this in other contexts, too. E.g. if a table uses a heading above the table, it would not be feasible to use a Group right?

A heading with right aligned Dropdown is quite a usual pattern in ILIAS. And I think this should be centralized and reused in these cases.

Of course the group could now use a Heading component to render the heading itself. But either this would be Group internal instantiation of the Heading. The we need to pass heading properties through the group (the group should not have any knowledge about heading levels).

The other alternative would be that the consumer creates Heading instances and passes them to the groups. Kind of complicated, imo.